modules/@apostrophecms/ui/ui/apos/components/AposSpinner.vue

```diff
@@ -74,6 +89,14 @@ module.exports = {
 
       return 'color: var(--a-primary);';
     },
+    stopColorLight() {
+      const color = this.colorLight || this.color;
+      return color?.startsWith('--') ? `var(${color})` : color;
+    },
+    stopColorDark() {
+      const color = this.color === '--a-primary' ? '#6516dd' : this.color;
```

Comment:
@stuartromanek This is the other "old" color I'm directly using, does it have a CSS var analogue?